pkg/services/blockfetcher/blockfetcher.go

```diff
@@ -192,7 +192,9 @@ func New(chain Ledger, cfg config.NeoFSBlockFetcher, logger *zap.Logger, put fun
 
 func getHeaderSizeMap(chain config.Blockchain) map[int]int {
 	headerSizeMap := make(map[int]int)
-	headerSizeMap[0] = block.GetExpectedHeaderSize(chain.StateRootInHeader, chain.GetNumOfCNs(0))
+	// Genesis header size.
+	headerSizeMap[0] = block.GetExpectedHeaderSize(chain.StateRootInHeader, 0)
```

Comment:
```suggestion
	headerSizeMap[0] = block.GetExpectedHeaderSize(chain.StateRootInHeader, 0) 	// genesis header size.
```

Usually separate-line comments are related to the following *block* of code, and this particular comment is related to a single line.